nbconvert/exporters/tests/test_html.py

```diff
@@ -74,7 +74,7 @@ def test_prompt_number(self):
                 }
             }
         )
-        exporter = HTMLExporter(config=no_prompt_conf, template_file='full')    
+        exporter = HTMLExporter(config=no_prompt_conf, template_name='notebook')
```

Comment:
Maybe the name of the template should be `classic` instead of `notebook`

Reply:
For now 'default' is the classic one, it may have been a good idea to have the default html template be a 'nothing' template, without css and all
The classic template would just add some css to the template, have some opinion on the dom structure etc. This way the lab template would not inherit from the classical template.